tree/treeplayer/test/dataframe/dataframe_simple_tests.hxx

```diff
@@ -427,3 +427,22 @@ TEST(TEST_CATEGORY, Reduce)
    EXPECT_EQ(*r4, 24);
    ROOT::DisableImplicitMT();
 }
+
+TEST(TEST_CATEGORY, Accumulate)
+{
+   auto d = TDataFrame(5).DefineSlotEntry("x", [](unsigned int, ULong64_t e) { return static_cast<int>(e); });
+   // acc U(U,T), merge U(U,U), default initValue
+   auto r1 = d.Accumulate([](int x, int y) { return x + y; }, [](int x, int y) { return x + y; }, "x");
+   // acc U(U,T), merge U(U,U), initValue
+   auto r2 = d.Accumulate([](int x, int y) { return x * y; }, [](int x, int y) { return x * y; }, "x", 1);
+   EXPECT_EQ(*r1, 10);
+   EXPECT_EQ(*r2, 24);
+
+   ROOT::EnableImplicitMT(2);
+   auto d2 = TDataFrame(5).DefineSlotEntry("x", [](unsigned int, ULong64_t e) { return static_cast<int>(e); });
+   auto r3 = d2.Accumulate([](int x, int y) { return x + y; }, [](int x, int y) { return x + y; }, "x");
```

Comment:
Food for thought: what we've just discussed about Accumulate vs Aggregate. Spark has an aggregate function with the same signature. Accumulate could cause confusion with std::accumulate, which has a different purpose.